<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matchImage</class>
 <widget class="QDialog" name="matchImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1611</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>matchImage</string>
  </property>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="viewImage1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>600</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>100</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QRadioButton" name="chk_newImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load new image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chk_crop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop from input image </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Step1: Input Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="viewImage2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>80</y>
     <width>341</width>
     <height>651</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QTreeWidget" name="result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>301</width>
      <height>601</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_viewImg1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>710</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>View (x,y)</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>650</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_match1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Match</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>350</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Details</string>
   </property>
   <widget class="QLineEdit" name="coef">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>coefficient(0.5 - 1.0):</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_Flip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>710</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>350</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Medthod</string>
   </property>
   <widget class="QRadioButton" name="chk_correlation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>81</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Correlation-Base</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="chk_shape">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape-Base</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Step2: Input Template</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>300</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step3: Select Match Medthod&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="viewImage3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>570</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>550</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Interested location Image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>590</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Brightness</string>
   </property>
   <widget class="QSlider" name="brightnessResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>255</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>180</string>
    </property>
   </widget>
  </widget>
  <zorder>listView</zorder>
  <zorder>listView_2</zorder>
  <zorder>viewImage1</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>label_2</zorder>
  <zorder>viewImage2</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>btn_viewImg1</zorder>
  <zorder>btn_match1</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>btn_load</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>listView_3</zorder>
  <zorder>viewImage3</zorder>
  <zorder>label_12</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>brightnessResult</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1065</x>
     <y>663</y>
    </hint>
    <hint type="destinationlabel">
     <x>1065</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
